WcaOnRails/config/locales/fr.yml

```diff
@@ -353,7 +353,7 @@ fr:
         generate_website: >-
           Je veux utiliser le site de la WCA plutôt qu'un site externe pour les
           informations complémentaires concernant la compétition.
-        #original_hash: dd88e47
+        #original_hash: 6cf623b
```

Comment:
I know there are quite a lot of these "original_hash" updates only, it's just that the changes were mainly about "entry fee" being turned into "registration fee", but the word "registration" was what we were already using in French.